site/content/api/Command-And-Control-API.md

```diff
@@ -3,140 +3,106 @@ title = "Command & Control API"
 
-| Processing in protocol adapter                         | AMQP outcome                  |
-|--------------------------------------------------------|------------------------------ |
-| Bad command                                            | Rejected ("hono:bad-request") |
-| Device not connected (anymore)                         | Release                       | 
-| Command approved (will be tried to deliver to device)  | Accepted                      | 
+**Message Format**
 
-Depending on the strategy for the receiver link address (see [Strategy for building the receiver link address]({{< relref "#choosing-a-strategy-for-building-the-receiver-link-address">}}) ), the *Business application* may want to close the response receiver
-link again if the outcome was not set to `Accepted`.
+The following table provides an overview of the properties the *Business Application* needs to set on a notification command message.
 
+| Name             | Mandatory | Location                 | Type         | Description |
+| :--------------- | :-------: | :----------------------- | :----------- | :---------- |
+| *subject*        | yes       | *properties*             | *string*     | The name of the command to be executed by the device. |
+| *content-type*   | no        | *properties*             | *string*     | If present, MUST contain a *Media Type* as defined by [RFC 2046](https://tools.ietf.org/html/rfc2046) which describes the semantics and format of the command's input data contained in the message payload. However, not all protocol adapters will support this property as not all transport protocols provide means to convey this information, e.g. MQTT 3.1.1 has no notion of message headers. |
+| *message-id*     | yes       | *properties*             | *string*     | An identifier that uniquely identifies the message at the sender side. |
 
-# Operations
+The command message MAY contain arbitrary payload to be sent to the device in a single AMQP *Data* section. The value of the message's *subject* property may provide a hint to the device regarding the format, encoding and semantics of the payload data.
 
-The following API defines operations that can be used by *Business Applications* to send commands (which get a response 
-in return) or notification commands to devices over an AMQP 1.0 Network.
+Hono indicates the outcome of the operation by means of the following AMQP delivery states:
 
-## Send a Notification Command
+| Delivery State | Description                                      |
+| :------------- | :----------------------------------------------- |
+| `accepted`    | The command has been delivered to the device for processing. |
+| `released`    | The command has not been delivered to the device because the device is (currently) not connected. |
+| `rejected`    | The command has not been delivered to the device because it does not contain all required information. |
 
```

Comment:
I tend to agree but on the other hand, it is just a small table and I also like to have the information in place instead of having to click and jump to another place in the document from where you need to return then in order to continue reading. We also have two variants of the send command only so it doesn't seem to wasteful to have the same table in two places, does it?

Reply:
Accepted - I can live with the two duplicated, but small tables... :+1: